web/includes/generate_job.php

```diff
@@ -68,27 +73,29 @@
   $sizeStr = $sizeW . 'x' . $sizeH;
 
   # Calc encoder
-  if ($encoder == 'none') $encoderStr = "-c copy";
-  else if ($encoder == 'x264') $encoderStr = "-c:v libx264 -crf 23";
-  else if ($encoder == 'mpeg2') $encoderStr = "-c:v mpeg2video -qscale:v 5";
+  if ($encoder == 'none') $encoderStr = '-c copy';
+  else if ($encoder == 'x264') $encoderStr = '-c:v libx264 -crf 23';
+  else if ($encoder == 'mpeg2') $encoderStr = '-c:v mpeg2video -qscale:v 5';
   
   # Calc framerate
-  $framerateStr = $framerate / 4;
+  $framerateStr = $framerate / RATE_BASE;
+  # How to get actual framerate?
+  $hardCodedFramerate = framerateStr * 5;
 
   # Command lines
-  $commandline1 = "cd /mnt/rec/events/" . $event["MonitorId"] . "/" . $date  .  "/" . $eid;
-  $commandline2 = '/usr/bin/ffmpeg -y -framerate ' . $framerate . ' -i %05d-capture.jpg -s ' . $sizeStr . ' ' . 
-    $encoderStr . ' -pix_fmt yuvj420p "Event-_' . $eid . '-r' . $framerateStr  . '-s' . $size  . '.avi"';
+  $commandline1 = 'cd ' . ZM_DIR_EVENTS . '/' . $event['MonitorId'] . '/' . $date  .  '/' . $eid;
```

Comment:
We probably should have advised better on this.  ZM_DIR_EVENTS used to be the thing to do, but we can have multiple different storage locations.  The Event object has methods that can get you the correct path to the monitor. 
So you can do 
$Event = new Event($event); 
and then do 
= 'cd ' . $Event->Path();

Reply:
I think better approach would be to use function **createVideo** which generates videos using perl script zmvideo.pl